ui/app/AppLayouts/Profile/panels/CommunitiesListPanel.qml

```diff
@@ -8,88 +8,155 @@ import StatusQ.Controls 0.1
 import StatusQ.Popups 0.1
 
 import utils 1.0
-
 import shared.controls.chat.menuItems 1.0
 
 StatusListView {
     id: root
 
-    property bool hasAddedContacts: false
+    property var rootStore
 
     signal inviteFriends(var communityData)
 
     signal closeCommunityClicked(string communityId)
     signal leaveCommunityClicked(string community, string communityId, string outroMessage)
-
     signal setCommunityMutedClicked(string communityId, int mutedType)
-
     signal setActiveCommunityClicked(string communityId)
+    signal showCommunityIntroDialog(string communityId, string name, string introMessage, string imageSrc, int accessType)
+    signal cancelMembershipRequest(string communityId)
```

Comment:
By name, it appears as a function from the client's side. It would be great if this followed the Qt signal naming convention.

Reply:
Agree... but existing code :)